Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3088,7 +3088,7 @@ IsLoggedInAPIEnabled:
 # FIXME: This is not implemented for WebKitLegacy, so should be excluded from WebKitLegacy entirely.
 IsNSURLSessionWebSocketEnabled:
   type: bool
-  status: developer
+  status: stable
```

Comment:
Shouldn't this be "shipping" ?

Reply:
I think we should move to `shipping` for our next major Safari release, but not for a SU.